test/torch/tensors/test_large_precision.py

```diff
@@ -233,8 +233,21 @@ def test_mod():
         (torch.tensor([2000.0123458910]), torch.tensor([2000.0123458910])),
     ],
 )
-def test_types(x, expected):
-    enlarged = x.fix_prec(internal_type=torch.int16, precision_fractional=128)
-    restored = enlarged.float_precision()
-    # And now x and restored must be the same
-    assert torch.all(torch.eq(expected, restored))
+class TestParametrized:
```

Comment:
Why is it needed to create a class?
Maybe it's a good practice, but we haven't done it so far, so I'm just curious :)

Reply:
It makes it easier to share the parameters for the tests enclosed in the class.
Happy to change it for something else

Reply:
All good, but I wonder if we shouldn't do that for more test actually

Reply:
It might be easier to put the test data in a var and pass it to the parametrize annotation.
```
test_data = [
    (torch.tensor([1]), torch.tensor([1.0])),
    (torch.tensor([1.0]), torch.tensor([1.0]))
]

@pytest.mark.parametrize("x, expected", test_data)
def test_types(x, expected):
    ...
```

Reply:
Yes, this looks like a good idea!